wordpress-mobile/WordPress-iOS
# Update media picker to version 0.11

This updates bring the capability to preview assets on the media picker.

To test:
 - Open WordPress app
 - Create a new post
 - Tap on add media button on the editor
 - Long Press/and or force touch on assets
 - Check if the force touch preview and/or full screen preview of asset shows
 - Make sure to test the following combinations:
  - WordPress Media Library from WP.com site and WP.org sites (self-hosted)
  - Images and videos
  - Local Device media

Needs review: @frosty
